### ISSUE TITLE
Cannot comment anymore

### ISSUE THREAD

**Issue by 0xABAB** [NixOS/nixpkgs#27244]
## Issue description
GitHub says "You can't comment at this time.".

### Steps to reproduce
User grahamc has put me on his personal block list, which resulted in me not being able to comment on https://github.com/NixOS/nixpkgs/issues/18223. This feature is documented at https://help.github.com/articles/blocking-a-user-from-your-personal-account/.

**Comment by vcunat** [comment]
I don't think so.  The conversation is unlocked and I haven't seen ability to block particular contributor-thread pairs.  Sometimes I got such errors transiently due to network failures.  Still, there might be stuff I've missed...

**Comment by 0xABAB** [comment]
@vcunat I just tried in another random thread opened by @grahamc and I suspect he has me on his blocked users list. I didn't know that was how the feature worked, but I suspect that's how it works.

**Comment by 0xABAB** [comment]
@vcunat As a consequence, he would also not see my @grahamc. So, if someone could ask him to reconsider would be swell.

**Comment by vcunat** [comment]
Ah, yes, [that's probably what's happened](https://help.github.com/articles/blocking-a-user-from-your-personal-account/).

**Comment by Profpatsch** [comment]
@0xABAB, you have been [blocked from the organization](https://github.com/NixOS/nixpkgs/issues/27258#issuecomment-314057285), due to your hostile behaviour towards contributors.

**Comment by danbst** [comment]
@edolstra IMO @0xABAB should be unbanned. He/she/they has a really strange and trollish way of talking, but there should be some warnings first, before ban. Perhaps he/she/they were warned already?

@Profpatsch commenting on your opinion:
> the person suddenly appeared a few days ago (see their profile) and started commenting hundreds of issues/PRs, always with a doubtful attitude
that looks... great! I've looked through some of his reviews, and all of them were on-topic. 

The way I see it, @0xABAB has a constant 1y+ interest in Nixpkgs/NixOS projects and wants them to be improved, just like me.

**Comment by vcunat** [comment]
I'd personally think that raising this ticket and finding to be banned by an individual member should have been a warning by itself, and a day or two after that there was that "final" thread...

**Comment by grahamc** [comment]
I added this user to my personal block list almost exactly a year ago (2016-06-07), where I asked another person in the community to block them. At that time, I noted that they had recently been blocked by other major projects as well.

At that time, I asked the other community member I messaged to review some other threads this user had been part of. This resulted in those threads being updated, to remove the nastiness.

In my opinion, the most valuable thing we have put together is our welcoming and supportive community. Anything that jeopardizes that jeopardizes the future of NixOS. I don't volunteer my time here to be berated, and I don't think anyone else does, either. I love this community, I deeply appreciate all the hard work from all the hundreds of contributors, and absolutely support @edolstra's decision to remove people who make it unpleasant to be part of an otherwise top notch group.
